src/Aspire.Hosting.Milvus/MilvusServerResource.cs

```diff
@@ -40,7 +40,7 @@ public MilvusServerResource(string name, ParameterResource apiKey) : base(name)
     /// </summary>
     public ReferenceExpression ConnectionStringExpression =>
        ReferenceExpression.Create(
-            $"Endpoint={PrimaryEndpoint.Property(EndpointProperty.Url)};Key={ApiKeyParameter}");
+            $"Endpoint={PrimaryEndpoint.Property(EndpointProperty.Url)};Key=root:{ApiKeyParameter}");
```

Comment:
When someone goes to deploy their app, are there Milvus endpoints they can point to? I'm wondering how the `root` gets replaced to some other user.

Reply:
We have a similar pattern for MySql https://github.com/dotnet/aspire/blob/ba949b9ca1fa2379e83da648a3e5c822fdce1437/src/Aspire.Hosting.MySql/MySqlServerResource.cs#L41 -- I assume the answer here would be they would use the Client config and set the Key param there more specifically. Similarly like MySql, Milvus could be configured to add more users with roles, etc. and then specify the desired key for actions. However, this is only configurable post startup in Milvus. root is always defined initially.